openid-provider-commands-1_0.md

```diff
@@ -385,53 +393,73 @@ Following is a non-normative response to an unsuccessful Maintain Command for a
 ```
 
 
+## Asynchronous Response
+
+When an RP is provided with a valid Asynchronous Command and the account is in a compatible state, the RP returns the HTTP 202 Accepted response.  When the Asynchronous Command has completed processing, and if the OP provided a `callback_endpoint` in its metadata and a `callback_token` in the command, the RP MUST do an HTTP POST of the result of processing to the `callback_endpoint` and include the `callback_token` as a Bearer token in the HTTP `Authorize` header.
```

Comment:
What is meant by `compatible state` ?

Reply:
- compatible state -- per the ISO state diagram, some transitions are not allowed 

- my thinking is that the `callback_token` is the only way client authentication happens to keep things simple. The RP is going to need some mechanism to provide context to the OP when sending an async response to a command -- so if we also supported other ways of client authentication, then we would need to come up with some other mechanism to provide context for the OP to know which async response this is.